### PR TITLE
Separate Default pages from Custom pages

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

File: src/app/models/page.ts [hunk 1]
```diff
@@ -5,4 +5,13 @@ export class Page extends AbstractPage {
   filename: string;
   resource: Resource;
   position: number;
+
+  static compare(a: Page, b: Page): number {
+    if (a.position < b.position) {
```
Comment (OzzieOrca) [inline src/app/models/page.ts]: I think you could just do `return a.position - b.position;`.

Comment (frett) [reply src/app/models/page.ts]: I like this, I don't think I've ever thought of that before

**Comment by frett** [comment]
I'll look into the lint errors tomorrow. I'd like to keep the CI server happy if possible :)

File: src/app/models/page.ts (hunk 1 above)
Comment (OzzieOrca) [reply src/app/models/page.ts]: `return a.attr.localeCompare(b.attr);` is also super nice if you are dealing with strings (which you aren't in this case).

**Pull request merged**